extended/src/test/java/io/kubernetes/client/extended/kubectl/KubectlLabelTest.java

```diff
@@ -13,7 +13,6 @@
 package io.kubernetes.client.extended.kubectl;
 
 import static com.github.tomakehurst.wiremock.client.WireMock.*;
```

Comment:
Can you remove this wildcard import too and replace it with specific imports (do this in all files)

Reply:
Done.